src/apps/logging/logging.cpp

```diff
@@ -342,6 +342,7 @@ namespace loggingapp
         ccf::historical::adapter(
           get_historical, context.get_historical_state(), is_tx_committed))
         .set_auto_schema<LoggingGetHistorical>()
+        .set_forwarding_required(ccf::ForwardingRequired::Never)
```

Comment:
Even though the historical query endpoint is `HTTP_GET`, it is created with `make_endpoint()` (as opposed to `make_read_only_endpoint()`) so the command was always be forwarded. We may want to revisit this in #1705.